cmd/server/main.go

```diff
@@ -33,15 +33,30 @@ const BindingGoogleKey = "googleKey"
 const BindingProjectNumber = "projectNumber"
 
 const UpsSecretName = "unified-push-server"
+const UpsURI = "uri"
 const GoogleKey = "googleKey"
 
 // This is required because importing core/v1/Secret leads to a double import and redefinition
 // of log_dir
 type BindingSecret struct {
-	metav1.TypeMeta              `json:",inline"`
-	metav1.ObjectMeta            `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
-	Data       map[string][]byte `json:"data,omitempty" protobuf:"bytes,2,rep,name=data"`
-	StringData map[string]string `json:"stringData,omitempty" protobuf:"bytes,4,rep,name=stringData"`
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
+	Data              map[string][]byte `json:"data,omitempty" protobuf:"bytes,2,rep,name=data"`
+	StringData        map[string]string `json:"stringData,omitempty" protobuf:"bytes,4,rep,name=stringData"`
+}
+
+type UpsURLData struct {
+	baseURL string
+}
+
+//retrieve the UPS data (currently just uri)
+func getUPSData() *UpsURLData {
```

Comment:
Do you think it makes sense to do include that here: https://github.com/aerogear/ups-config-operator/blob/master/cmd/server/main.go#L231 ? (instead of an extra function)

Reply:
Yep makes sense - I will make an update